src/plugins/intel_cpu/src/serialize.cpp

```diff
@@ -61,6 +61,12 @@ void ModelDeserializer::operator>>(std::shared_ptr<ov::Model>& model) {
     StreamSerialize::DataHeader hdr = {};
     _istream.read(reinterpret_cast<char*>(&hdr), sizeof hdr);
 
+    // check if model header contains valid data
+    bool isValidModel = (hdr.custom_data_size == hdr.consts_offset - hdr.custom_data_offset) &&
```

Comment:
What I can suggest: 
- check value of hdr.custom_data_offset == sizeof(hdr);
- not sure if we can check full size of _istream (compare it with hdr.cusotm_data/consts/model_size + sizeof(hdr));

Reply:
Thanks for your advice, I also consider check the full size but the C++ stream API is not good at geting the total length.

Reply:
Both advice is implemented in dc9d82fb0d1fdaa9dee83717d4c8dc1b0b0fbf94